config/environments/development.rb

```diff
@@ -9,6 +9,15 @@
   config.cache_classes = false
   config.eager_load = false
 
+  # Guard's 'listen' gem is measurably faster than filesystem polling
+  # but some people have a bad time with it.  Opt in if you already
+  # installed it (e.g. in Gemfile.dev.rb).
+  begin
+    require 'listen'
+    config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  rescue LoadError
```

Comment:
I'm unsure how we either let people know this is an option and/or tell them which type of  file_watcher we're currently using.

While I agree that the inotify/listen route is probably much better on linux, it's hard to suggest it if it's so bad on OSX.

I feel like we should suggest it on linux.

At the same time, I feel like decreasing the frequency of the poll on all platforms would be a good change to rails that would mitigate the need for the listen gem.